langchain4j-anthropic/src/main/java/dev/langchain4j/model/anthropic/internal/client/DefaultAnthropicClient.java

```diff
@@ -15,29 +16,35 @@
 import org.slf4j.LoggerFactory;
 import retrofit2.Call;
 import retrofit2.Retrofit;
-import retrofit2.converter.gson.GsonConverterFactory;
+import retrofit2.converter.jackson.JacksonConverterFactory;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicInteger;
 
-import static com.google.gson.FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES;
-import static com.google.gson.ToNumberPolicy.LONG_OR_DOUBLE;
+import static com.fasterxml.jackson.databind.PropertyNamingStrategies.SNAKE_CASE;
+import static com.fasterxml.jackson.databind.SerializationFeature.INDENT_OUTPUT;
+import static com.fasterxml.jackson.databind.cfg.EnumFeature.WRITE_ENUMS_TO_LOWERCASE;
 import static dev.langchain4j.internal.Utils.*;
 import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
-import static dev.langchain4j.model.anthropic.AnthropicMapper.toFinishReason;
+import static dev.langchain4j.model.anthropic.internal.mapper.AnthropicMapper.toFinishReason;
 import static java.util.Collections.synchronizedList;
 
 public class DefaultAnthropicClient extends AnthropicClient {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultAnthropicClient.class);
 
-    static final Gson GSON = new GsonBuilder()
-            .setFieldNamingPolicy(LOWER_CASE_WITH_UNDERSCORES)
-            .setObjectToNumberStrategy(LONG_OR_DOUBLE)
-            .setPrettyPrinting()
-            .create();
+    private static final ObjectMapper OBJECT_MAPPER = createObjectMapper();
+
+    private static ObjectMapper createObjectMapper() {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.setPropertyNamingStrategy(SNAKE_CASE);
```

Comment:
@geoand in Quarkus extension mixins are used to define (de)serialization stratigies, can you switch to the global settings on `ObjectMapper` level? Or should I annotate every api model class instead?

Reply:
The reason I used mixins was because I could not change the classes themselves :). 

Most of the uses of mixins would be hard to replicate on the `ObjectMapper` configuration

Reply:
As far as I understand, `AnthropicContentMixin` and `AnthropicContentMixinBuilderMixin` can be removed as I removed builder from `AnthropicContent` in this PR.

`AnthropicMessageMixin` can probably be also removed as I have made `AnthropicMessage`'s fields public in this PR and added default ctor.
`AnthropicMessageSerializer` seems to be not used.

And `AnthropicRoleDeserializer`, `AnthropicRoleMixin` and `AnthropicRoleSerializer` can probably be replaced with `objectMapper.configure(WRITE_ENUMS_TO_LOWERCASE, true);`.

So then all mixins can be removed I guess?

Reply:
Yeah, that likely can with these changes